src/transformers/models/bert/modeling_tf_bert.py

```diff
@@ -659,18 +714,22 @@ class PreTrainedModel
 
     def call(
         self,
-        input_ids=None,
-        attention_mask=None,
-        token_type_ids=None,
-        position_ids=None,
-        head_mask=None,
-        inputs_embeds=None,
-        output_attentions=None,
-        output_hidden_states=None,
-        return_dict=None,
-        training=False,
+        input_ids: Optional[
+            Union[
+                List[tf.Tensor], List[np.ndarray], Dict[str, tf.Tensor], Dict[str, np.ndarray], np.ndarray, tf.Tensor
+            ]
+        ] = None,
```

Comment:
This is the point where the bare type annotation is not useful. We should define a type `TFModelInputType` (or an other better name) to avoid that takes everything in the Union so it can be more readable.